### ISSUE TITLE
Make the TOML LSP available as a standalone package?

### ISSUE THREAD

**Issue by bolinfest** [bungcip/better-toml#13]
It appears that `out/server/tomlServerMain.js` can function as a standalone LSP. Would you be interested in releasing it as such? Then it could be used in other editors that speak LSP, as well.

**Comment by bungcip** [comment]
yes i interested to package the server as standalone LSP. It's low priority though. Current priority is to change parser in the extension from `node-toml` to `degurechaff`. I will revisit this issue once the move is done.
